packages/schematics/angular/ng-new/schema.json

```diff
@@ -125,10 +125,11 @@
         "message": "Which stylesheet format would you like to use?",
         "type": "list",
         "items": [
-          { "value": "css",  "label": "CSS    (.css )" },
-          { "value": "sass", "label": "Sass   (.scss) [ http://sass-lang.com   ]" },
-          { "value": "less", "label": "Less   (.less) [ http://lesscss.org     ]" },
-          { "value": "styl", "label": "Stylus (.styl) [ http://stylus-lang.com ]" }
+          { "value": "css",  "label": "CSS" },
+          { "value": "scss", "label": "SCSS   [ http://sass-lang.com   ]" },
```

Comment:
Note: Unlike Sass, SCSS is always written in all caps

Reply:
`Sass SCSS`/`Sass Indented`?

Reply:
Is the name `Sass Indented` or `Indented Sass`?

Reply:
`Sass Indented`
https://sass-lang.com/documentation/file.INDENTED_SYNTAX.html

Reply:
I suppose we could ask for clarification on full official naming.

Reply:
Sass's indented syntax (also known simply as "Sass”)

Reply:
I'm fine with it either way.  But I think we just need a general consensus so this isn't revisited in another 2 weeks.

Reply:
In their docs to be clear they mostly use the syntax kind as references ie Sass and SCSS

Reply:
I think the links here should go all the way to the docpage https://sass-lang.com/documentation/file.INDENTED_SYNTAX.html

Somehow users need to know more to pick the right thing from the list, most of them are probably not aware there are multiple syntax nor how to pick between them

AS much as we can disambiguate in the label, that's also good. Maybe

`SCSS (CSS syntax)  [http://sass-lang.com/documentation/file.SASS_REFERENCE.html#syntax]`
`Sass (intented syntax) [https://sass-lang.com/documentation/file.INDENTED_SYNTAX.html]`


either way I don't care to revisit this :)

Reply:
@alexeagle I think you meant `Sass (SCSS syntax)`?

I think in general and this was also how it was before, SCSS refers to the SCSS syntax and Sass refers to the indented syntax.

My point is that Sass & SCSS are the most commonly used terms, and for novice users putting other terms might cause more confusion then good.

In most cases, for example, in Sass documentation, when they refer to Sass, they refer to the indented syntax and not the pre-processor.

https://sass-lang.com/documentation/file.SASS_REFERENCE.html#syntax_selection
> SCSS syntax works straightforwardly in Sass by using newlines instead of semicolons and indentation instead of braces.